src/engine/renderer/tr_init.cpp

```diff
@@ -256,12 +256,15 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 	Cvar::Cvar<bool> r_materialDebug( "r_materialDebug", "Enable material debug SSBO", Cvar::NONE, false );
 	cvar_t      *r_showParallelShadowSplits;
 
-	Cvar::Cvar<bool> r_profilerRenderSubGroups( "r_profilerRenderSubGroups", "Enable subgroup profiling in rendering shaders", Cvar::NONE, false );
+	Cvar::Cvar<bool> r_profilerRenderSubGroups( "r_profilerRenderSubGroups", "Enable subgroup profiling in rendering shaders", Cvar::CHEAT, false );
 	Cvar::Range<Cvar::Cvar<int>> r_profilerRenderSubGroupsMode( "r_profilerRenderSubGroupsMode", "Red: more wasted lanes, green: less wasted lanes; "
 		"0: VS opaque, 1: VS transparent, 2: VS all, 3: FS opaque, 4: FS transparent, 5: FS all", Cvar::NONE,
 		Util::ordinal( shaderProfilerRenderSubGroupsMode::VS_OPAQUE ),
 		Util::ordinal( shaderProfilerRenderSubGroupsMode::VS_OPAQUE ),
 		Util::ordinal( shaderProfilerRenderSubGroupsMode::FS_ALL ) );
+	Cvar::Cvar<int> r_profilerRenderSubGroupsStage( "r_profilerRenderSubGroupsStage", "Select a specific"
+		"stage/material ID (if material system is enabled) for subgroup profiling (-1 to always profile the last stage/material); "
```

Comment:
Should this say 'all materials' instead of 'last material'?

Reply:
I've changed this to say "-1 to profile all stages/materials, rendered in theuir usual order" now.

Reply:
That doesn't match the non-material behavior now which only does the last stage with -1.

Reply:
Right, I changed stage behaviour to match now.